# mission poses
mission:
  gate_position: [0.0, 0.0, 1.2]
  gate_ori_euler: [0.0, 0.0, 0.0]

  goal_position: [0.0, 0.0, 1.2]
  mission/goal_ori_euler: [0.0, 0.0, 0.0]


# flight mission
traj_server:
  takeoff_height: 1.2

# mpc cost weights
learning_agile:
  max_traverse_weight: 70